src/vivarium_ciff_sam/constants/data_values.py

```diff
@@ -1,34 +1,13 @@
-from datetime import datetime
```

Comment:
Removed these placeholders that came from the template. Maybe they shouldn't be in the template at all?